types/src/package.rs

```diff
@@ -29,7 +29,7 @@ use crate::{
     AddressableEntityHash, CLType, CLTyped, HashAddr, BLAKE2B_DIGEST_LENGTH, KEY_HASH_LENGTH,
 };
 
-const PACKAGE_STRING_PREFIX: &str = "contract-package-";
+const PACKAGE_STRING_PREFIX: &str = "package-";
 // We need to support the legacy prefix of "contract-package-wasm".
 const PACKAGE_STRING_LEGACY_EXTRA_PREFIX: &str = "wasm";
```

Comment:
we should be able to delete PACKAGE_STRING_LEGACY_EXTRA_PREFIX because this a all new type, completely distinct from CONTRACT_PACKAGE and its legacy / bugged wasm prefix handling

Reply:
Got rid of the constant  by moving into where ContractPackageHash lives